src/tools/jsondoclint/src/validator.rs

```diff
@@ -221,7 +221,7 @@ impl<'a> Validator<'a> {
         }
     }
 
-    fn check_typedef(&mut self, x: &'a Typedef) {
+    fn check_typedef(&mut self, x: &'a TypeAlias) {
```

Comment:
NIT: Could you rename this to `check_typealias`

Reply:
Ditto, thanks.